### ISSUE TITLE
Multi-dash is not aligned vertically with regular dash

### ISSUE THREAD

**Issue by sabi0** [tonsky/FiraCode#429]
![image](https://cloud.githubusercontent.com/assets/11509875/26619759/25bf6894-45e0-11e7-936b-9618695e5cd3.png)

**Comment by tonsky** [comment]
Yeah, I know why it happens. I’ll take a look eventually, thanks :)
